python/valis/db/queries.py

```diff
@@ -690,8 +690,39 @@ def get_catalog_sources(sdss_id: int) -> peewee.ModelSelect:
     """
 
     s = vizdb.SDSSidFlat.select(vizdb.SDSSidFlat).where(vizdb.SDSSidFlat.sdss_id == sdss_id).alias('s')
-    return cat.Catalog.select(cat.Catalog, starfields(s)).\
-        join(s, on=(s.c.catalogid == cat.Catalog.catalogid)).order_by(cat.Catalog.version.desc())
+    return cat.Catalog.select(cat.Catalog, cat.SDSS_ID_To_Catalog, starfields(s)).\
+        join(s, on=(s.c.catalogid == cat.Catalog.catalogid)).\
+        join(cat.SDSS_ID_To_Catalog, on=(s.c.catalogid == cat.SDSS_ID_To_Catalog.catalogid)).\
+        order_by(cat.Catalog.version.desc())
+
+
+def get_parent_catalogs(catalogid: int | Sequence[int]) -> peewee.ModelSelect:
+    """ Get the parent catalog info for a target catalogid
+
```

Comment:
This is no longer being used.  Do we want to keep this around as a useful query?

Reply:
Yeah, this is no longer used. Let's delete it; it's easy to add back if needed and I've added a similar function for #33